deepcell_label/models.py

```diff
@@ -117,7 +116,6 @@ class Project(db.Model):
     channel = db.Column(db.Integer, default=0)
     feature = db.Column(db.Integer, default=0)
     scale_factor = db.Column(db.Float, default=1)
-    colormap = db.Column(db.PickleType)
 
```

Comment:
Does removing fields cause any issues with the schema difference?

Or does it just get ignored bc SQL doesn't include the field in the queries?

Reply:
Dropping a column should be fine. The old column will still be in our production database, but it will be ignored and unused. Old rows will still have data in the column while new rows will not write to the column, and we can't access the column through SQLAlchemy because it's no longer in the schema